### ISSUE TITLE
SHIFT does not work to select text via SHIFT+Arrow Keys

### ISSUE THREAD

**Issue by kladderadeng** [Ylianst/MeshCentralRouter#104]
Describe the bug
When controlling a remote Windows 11 24H2 system, being in any kind of Text window (like Notepad), you can not select text via SHIFT+Arrow keys. The SHIFT key itself works as you can type upper case letters, but you can not select text with it.
It does not matter whether I have WebRTC enabled or disabled or if I use the "use remote keyboard layout" function or not.

To Reproduce

Remote access another Windows 11 24H2 System via the Windows Router executable or the Web Client.
Type something and then try to select it with SHIFT+Arrowkeys. The cursor will move, but nothing will be elected.

Expected behavior
Since the SHIFT key works, I would expect, that SHIFT+Arrowkeys works also.

Server Software (please complete the following information):

OS: Ubuntu Server 22.04.5 LTS
Virtualization: Docker Image ghcr.io/ylianst/meshcentral:1.1.32
Network: Both testsystems are on the same Network (LAN), but Meshcentral runs through Traefik and I am using Hybrid mode
Version: 1.1.32
Node: 20.15.1
Client Device (please complete the following information):

Device: PC
OS: Windows 11 23H2
Network: LAN (Source PC) to VLAN (Traefik + Meshcentral) to LAN (Destination PC)
Browser: Microsoft Edge and Vivaldi
MeshCentralRouter Version: 1.8.9021
Remote Device (please complete the following information):

Device: VM under XCP-ng
OS: Windows 11 24H2
Network: LAN
Current Core Version: Dec 9 2022, 1874374695
Additional context

Your config.json file

{
  "$schema": "https://raw.githubusercontent.com/Ylianst/MeshCentral/master/meshcentral-config-schema.json",
  "settings": {
    "Cert": "meshcentral.my.domain",
    "plugins":{"enabled": false},
    "_mongoDb": null,
    "cert": "localhost",
    "_WANonly": true,
    "_LANonly": true,
    "_sessionKey": "redacted",
    "port": 443,
    "aliasPort": 443,
    "redirPort": 80,
    "redirAliasPort": 80,
    "AgentPong": 300,
    "TLSOffload": true,
    "SelfUpdate": false,
    "AllowFraming": false,
    "Minify": 1,
    "WebRTC": true
  },
  "domains": {
    "": {
      "title": "MyLab",
      "title2": "Meshcentral",
      "minify": true,
      "NewAccounts": false,
      "localSessionRecording": false,
      "_userNameIsEmail": true,
      "certUrl": "meshcentral.my.domain",
      "allowedOrigin": [ "meshcentral.my.domain" ]
    }
  },
  "_letsencrypt": {
    "__comment__": "Requires NodeJS 8.x or better, Go to https://letsdebug.net/ first before>",
    "_email": "myemail@mydomain.com",
    "_names": "myserver.mydomain.com",
    "production": false
  }
}

**Comment by si458** [comment]
Should fixed by the recent commit, so we will get builds made and released for next month!

**Comment by si458** [comment]
yep recent commit/pr fixes the issue https://github.com/Ylianst/MeshCentralRouter/pull/122
new build will be incoming in next months release 👍
